DEVELOPMENT.md

```diff
@@ -94,3 +94,22 @@ Then `package.json` should have the same versions specified as well:
 ```
 
 If you have a mismatch in `package.json`, say `fhir-works-on-aws-routing` was set to `4.1.0` instead of `4.0.0`. An error message
+
+### Accessing Logs and Debugging on FWoA
+Logs and Debugging are handled by [CloudWatch](https://docs.aws.amazon.com/cloudwatch/index.html). Detailed instructions on customizing CloudWatch to organize troubleshooting can be found [here](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/WhatIsCloudWatch.html).
+
+To access the logs generated by FWoA:
+1. Log on to the AWS Console.
+2. From the list of AWS services, select **CloudWatch**.
+3. From the tab on the left, select **Log Groups**
+4. From the list of log groups, select the Lambda Function or the APIGateway service that you would like to debug.
+  * `/aws/api-gateway/fhir-service-{STAGE}` contains all the logs related to the setup of the API Gateway for FWoA.
+  * `/aws/lambda/fhir-service-{STAGE}-ddbtoES` contains all the logs from the process of writing DynamoDB resources to ElasticSearch.
+  * `/aws/lambda/fhir-service-{STAGE}-fhirServer` contains all the logs from the FWoA API server, inclusing transactions and requests. This will be the log group to check when receiving errors from the FWoA server, in most cases.
+  * `/aws/lambda/fhir-service-{STAGE}-getJobStatus` contains all the logs related to querying the status of a Bulk Export job.
+  * `/aws/lambda/fhir-service-{STAGE}-startExportJob` contains all the logs related to starting a Bulk Export job.
+  * `/aws/lambda/fhir-service-{STAGE}-stopExportJob` contains all the logs related to stopping a Bulk Export job.
+  * `/aws/lambda/fhir-service-{STAGE}-updateStatus` contains all the logs related to updating a Bulk Export job.
```

Comment:
Another one here `/aws-glue/jobs/output` contains all the logs related to the Bulk Export Glue job execution.